plugins/com.google.cloud.tools.eclipse.dataflow.ui/src/com/google/cloud/tools/eclipse/dataflow/ui/handler/ModifyDataflowVersionHandler.java

```diff
@@ -64,8 +57,12 @@ public Object execute(ExecutionEvent event) throws ExecutionException {
     if (Strings.isNullOrEmpty(commandId)) {
       return null;
     }
-    ISelection selection = HandlerUtil.getCurrentSelection(event);
-    IProject project = getFirstProjectInSelection(selection);
+    IProject project = ProjectFromSelectionHelper.getFirstProject(event);
+    if (project == null) {
+      throw new ExecutionException(
+          "Exception while modifying Dataflow Maven Dependency: no project selected");
```

Comment:
This exception message comes from the code a bit below in this method. That said, it basically throws the same exception as before if `project` is `null`.